clients/admin-ui/src/types/api/models/ExperienceTranslation.ts

```diff
@@ -57,4 +57,8 @@ export type ExperienceTranslation = {
    * Overall description - used for banner as well if applicable.  HTML descriptions are supported so links can be included.
    */
   description?: string;
+  /**
+   * Custom link/button trigger label
+   */
+  trigger_link_label?: string;
```

Comment:
can we rename this across the board to `modal_link_label`? This is clearer that the link opens the modal, kind of analogous to how we name the `privacy_policy_link_label`, where this link opens the privacy policy.

Reply:
love it.